epoch_multinode_test.go

```diff
@@ -271,6 +295,23 @@ func (n *inMemNetwork) addNode(node *testNode) {
 	n.instances = append(n.instances, node)
 }
 
+func (n *inMemNetwork) IsConnected(node NodeID) bool {
+	for _, instance := range n.instances {
```

Comment:
Not sure I follow - why we're iterating over all instances, can't we just do a map lookup?